spine-csharp/src/SkeletonBinary.cs

```diff
@@ -40,11 +44,12 @@
 
 namespace Spine {
 	public class SkeletonBinary {
-		public const int TIMELINE_SCALE = 0;
-		public const int TIMELINE_ROTATE = 1;
-		public const int TIMELINE_TRANSLATE = 2;
-		public const int TIMELINE_ATTACHMENT = 3;
-		public const int TIMELINE_COLOR = 4;
+		public const int TIMELINE_ROTATE = 0;
```

Comment:
Any change (even just adding a value) to the binary format means previously exported data needs to be exported again, so changing the constants doesn't hurt. The change is to keep a consistent order for transform properties, always: rotate, translate, scale, then shear.